packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -848,7 +852,13 @@ export class IntervalCollectionIterator<TInterval extends ISerializableInterval>
     }
 }
 
-export class IntervalCollection<TInterval extends ISerializableInterval> extends EventEmitter {
+export interface IIntervalCollectionEvent<TInterval extends ISerializableInterval> extends ISharedObjectEvents {
```

Comment:
should this extend ISharedObjectEvents? are all those events emitted? or should it jsut be IEvent

Reply:
Oh, let me check. IEvent might be fine.